lib/router/router.js

```diff
@@ -109,6 +126,17 @@ p.getInfo = function(req) {
     }
   }
 
+  // forward original image if no operation on image and browser support image format
+  if (routeInfo.imageSteps.length === 0 && (this.options.noop || routeInfo.urlInfo.query.noop)) {
```

Comment:
To clarify, is all you really need is to make `useOriginal` query option, a default? If so, `useOriginal` could be added into `options` as well. It should require minimal changes.